src/core/qfieldcloudconnection.h

```diff
@@ -144,7 +144,7 @@ class QFieldCloudConnection : public QObject
     /**
      * Returns the connections URLs successfully logged in in the past.
      */
-    Q_INVOKABLE QStringList urls() const;
+    QStringList urls() const;
```

Comment:
Why not invocable? If I remember right, we show the previous urls in the UI for people self-hosting?

Reply:
@suricactus , it's still there, as a Q_PROPERTY( QStringList urls READ urls ... ). I'm just removing the redundant Q_INVOKABLE here.